ember-flight-icons/tests/dummy/app/helpers/substring.js

```diff
@@ -0,0 +1,8 @@
+import Helper from '@ember/component/helper';
+
+export default class Substring extends Helper {
```

Comment:
suggestion: add a comment about the intended behavior of this helper

Reply:
ok, added 👍